js/src/lib/components/InferenceWidget/widgets/AudioToAudioWidget/AudioToAudioWidget.svelte

```diff
@@ -168,7 +170,7 @@
 >
 	<svelte:fragment slot="top">
 		<form>
-			<div class="flex flex-wrap items-center">
+			<div class="flex flex-wrap items-center {isDisabled ? 'pointer-events-none opacity-50' : ''}">
```

Comment:
I think we should also hide input controls as well when disabled

@julien-c do you have an opinion?

Reply:
yes that was the spirit of @gary149's Figma i think

Reply:
Ah just remembered this is included in the Figma

Reply:
> I think we should also hide input controls as well when disabled

Yes.

Reply:
handled in [here](https://github.com/huggingface/hub-docs/pull/1019#issuecomment-1787280284)